tests/test_join.py

```diff
@@ -105,3 +105,17 @@ def test_join_random(seed, lt):
     assert djoined.shape == (ndata, 2)
     assert djoined.names == ("KEY", "VAL")
     assert djoined.topython() == [main, res]
+
+
+
+def test_join_update():
+    d0 = dt.Frame([[1, 2, 3, 2, 3, 1, 3, 2, 2, 1], range(10)], names=("A", "B"))
+    d1 = d0[:, mean(f.B), f.A]
+    d1.key = "A"
+    d0[:, "AA", join(d1)] = g.V0
```

Comment:
`"AA"` here means that for `join` we're going to use a column `A` from `d0` with the column `A` from `d1`?

Reply:
Sorry, got it. This is just the name of the new column to be created...